resources/migrations/001_update_migrations.yaml

```diff
@@ -6114,6 +6114,21 @@ databaseChangeLog:
         - customChange:
             class: "metabase.db.custom_migrations.CreateInternalUser"
 
+  - changeSet:
+      id: v50.2024-03-29T10:00:00
+      author: piranha
+      comment: 'Granular cache invalidation'
+      changes:
+        - addColumn:
+            tableName: report_card
+            columns:
+              - column:
+                  name: cache_invalidated_at
+                  type: ${timestamp_type}
+                  remarks: 'If cache should be invalidated earlier than strategy expects it to'
```

Comment:
How about `an invalidation time that can supersede that on on the associated cache_config`?

"If" makes it sounds like a boolean, "expects" is strangely anthropomorphic, and technically speaking there's no guarantee this timestamp is earlier.

Reply:
done